README.rst

```diff
@@ -36,7 +36,14 @@ Installation
 
 Note that if you've already installed `mesos` locally, you can either install this to a location other than `/usr/local/bin` via. pip options or remove `/usr/local/bin/mesos`. There should be no downsides to just removing it.
 
-To build `mesos-cli`, run `python setup.py build` in the root of the repository. To install, run `python setup.py install`.
+
+To build, run ``python setup.py build`` in the root of the repository.
+
+To install, run ``python setup.py install``
```

Comment:
These are meant to be separate instructions, not ``make env`` requirements.  

I moved each instruction to be under separate headers to avoid confusion.